CONTRIBUTING.md

```diff
@@ -203,7 +203,7 @@ You will need a `./e2e/playwright/playwright-secrets.env` file:
 ```bash
 $ touch ./e2e/playwright/playwright-secrets.env
 $ cat ./e2e/playwright/playwright-secrets.env
-token=<dev.zoo.dev/account/api-tokens>
+token=<zoo.dev/account/api-tokens>
 snapshottoken=<zoo.dev/account/api-tokens>
```

Comment:
Should we just say dev token and prod token? At least this is how CI is setup. But I guess one can always run the test against prod or dev it desn't really matter

Reply:
@jacebrowning snapshottoken isn't even used anymore now that we removed the disabled test about export. this was just for zoo cli at the time. I think it's time we nuke this secrets file??

![image](https://github.com/user-attachments/assets/21f10285-69d9-4ed7-94bf-cfdad4b0e0cd)

Reply:
I think we should orient this doc to folks coming in from the internet. If we have separate internal dev process/servers those docs should be internal, ideally. So, we should scrub mentions of dev from this repo, I think. It'll never work for folks outside of our org.

Reply:
As discussed on slack I'll make a new issue!